### ISSUE TITLE
underflow in generated code

### ISSUE THREAD

**Issue by ghorn** [cvxgrp/cvxpygen#31]
I tried clarabel code generation and I got this error:
```
error: unsigned conversion from 'int' to 'long unsigned int' changes value from '-1' to '18446744073709551615' [-Werror=sign-conversion]
 5440 | cpg_csc acc_canon_A_map = {1802, 1802, 13, acc_canon_A_map_p, acc_canon_A_map_i, acc_canon_A_map_x, -1};
```

I'm compiling generated code myself, not using cvxpygen's built-in compilation, and have a lot of warning flags on. This one looked more suspicious than the others.

**Comment by ghorn** [comment]
Is `nz` from `struct cpg_csc` ever used? Can it be removed?

**Comment by ghorn** [comment]
PR to remove it is here https://github.com/cvxgrp/cvxpygen/pull/33

**Comment by maxschaller** [comment]
Thanks for pointing this out, closed by #33
